ground/src/main/java/com/google/android/ground/Config.kt

```diff
@@ -25,7 +25,7 @@ object Config {
   const val SHARED_PREFS_MODE = Context.MODE_PRIVATE
 
   // Local db settings.
-  const val DB_VERSION = 120
+  const val DB_VERSION = 121
```

Comment:
@gino-m Is it ok for destructive migration or should we start adding migrations to prevent data loss?

Reply:
We should aim to preserve data collected so far in the field, in which case we would want to implement migrations. But do we know if the user stays signed into the app after upgrading the app in Play Store?

Reply:
Not today, if there are any breaking changes in the db due to destructive migration.

Reply:
I would advise punting on this issue, saving the migration for when we switch over to serialized protos locally.

Reply:
Apologies that my comment on the sheet didn't take into consideration that the local db would need to be updated.